<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LWO1-DHP-F54947" name="IETest" tests="2" failures="2" timestamp="22 Aug 2014 10:08:54 GMT" time="149.073" errors="0">
  <testcase name="UserCanLogin" time="140.305" classname="InstabeatSanity.Tests.SanityTestNG">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at instabeat.pages.LoginPage.isUserLoggedIn(LoginPage.java:34)
	at InstabeatSanity.Tests.SanityTestNG.UserCanLogin(SanityTestNG.java:48)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- UserCanLogin -->
  <testcase name="UserCannotLogin" time="0.02" classname="InstabeatSanity.Tests.SanityTestNG">
    <failure type="java.lang.AssertionError" message="expected [] but found [Login]">
      <![CDATA[java.lang.AssertionError: expected [] but found [Login]
	at instabeat.pages.LoginPage.verifyPageTitle(LoginPage.java:23)
	at InstabeatSanity.Tests.SanityTestNG.UserCannotLogin(SanityTestNG.java:62)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- UserCannotLogin -->
</testsuite> <!-- IETest -->
